oneflow/python/nn/modules/acosh.py

```diff
@@ -50,12 +50,12 @@ def acosh_op(x):
         >>> flow.enable_eager_execution()
         >>> x1 = flow.Tensor(np.array([2, 3, 4]).astype(np.float32))
         >>> out1 = flow.acosh(x1)
-        >>> out1.numpy() #doctest: +ELLIPSIS
-        array([1.3169... , 1.7627..., 2.0634... ], dtype=float32)
+        >>> out1 #doctest: +ELLIPSIS
```

Comment:
```suggestion
        >>> out1
```
`#doctest: +ELLIPSIS` 可以去掉了，之前加这个是为了匹配4位小数后的数据，现在小数最多4位，没这个问题了。